Makefile

```diff
@@ -56,3 +56,11 @@ kafka-producer-run:
 				--bootstrap-server kafka:9092 \
 				--topic TestTopic \
 		"
+
+python-get-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-di/deployment)"
+	python application/get_message.py
+
+python-put-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-di/deployment)"
```

Comment:
Please, can we consider naming it `uec-dos-int-dev/deployment` to match our naming convention?